### ISSUE TITLE
Cannot clone submodule MPQExtractor

### ISSUE THREAD

**Issue by dawnstrider** [Sarjuuk/aowow#43]
I am trying to follow the instructions for the setup.

It says to 
```
git submodule update --init .... 
```

But I get an error:
```
Cloning into 'includes/tools/MPQExtractor'...
remote: Counting objects: 457, done.
remote: Total 457 (delta 0), reused 0 (delta 0), pack-reused 457
Receiving objects: 100% (457/457), 741.53 KiB, done.
Resolving deltas: 100% (144/144), done.
fatal: reference is not a tree: 32c72bb5f075f6f2ce93fa37a677de4cf507ed70
Unable to checkout '32c72bb5f075f6f2ce93fa37a677de4cf507ed70' in submodule path 'includes/tools/MPQExtractor'

```

**Comment by dawnstrider** [comment]
I was able to fix the issue on windows by following the comment of the commit

     https://github.com/Sarjuuk/aowow/commit/a2d4b014fc6575d8dc7e080db2c13b501f4c4654

I cannot fix this issue on a Linux env.

**Comment by Sarjuuk** [comment]
"fixed" in 5661f09454b2c21b4c1f7711b82e28e12da94dfd
submodules ... not even once!
